src/libmongoc/tests/test-mongoc-dns.c

```diff
@@ -441,7 +441,7 @@ test_all_spec_tests (TestSuite *suite)
    install_json_test_suite_with_check (
       suite,
       JSON_DIR,
-      "/initial_dns_seedlist_discovery/sharded",
+      "initial_dns_seedlist_discovery/sharded",
```

Comment:
This looks unrelated. Is it fixing an older issue not tracked by a ticket?

Reply:
I believe this is a just a change from a manual merge resolution, not a fix for any issue.